README.md

```diff
@@ -49,7 +49,7 @@ jobs:
     steps:
     - uses: actions/checkout@v3
       with:
-        ref: ${{ github.event.pull_request.head.sha }}
+        ref: ${{ github.sha }}
```

Comment:
Do we even need this? By default it checks out the current ref (branch)

Reply:
Good point as is just a readme example I'm happy with either leaving it or removing the ref: altogether. up to you XD